lib/tasks/generate_random_data.rake

```diff
@@ -1,31 +0,0 @@
-namespace :generate_random_data do
```

Comment:
@nronas Agree on not including this as part of unit test. But we should still keep it as a helper task(manual) to perf test in future.

Reply:
Everything else looks good apart from this, we can probably create a follow-up PR for this once we discuss. 

Merging this now to avoid re-work for frontend folks since this has API changes.